### PR TITLE
minor readme fix!

### PR DESCRIPTION
**Pull request by wgahnagl**
hello again! 
A lot has changed! I haven't had time to check if the fix for the editbox focus is working, but I'll be working on that this weekend!
here's a super quick fix for your readme that I noticed while I was going over it to see what I missed lol 

the webassembly stuff looks Extremely cool!

### CODE REVIEW

**Comment by cschreib** [comment]
Damn it, thank for noticing!

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ brew install sfml zlib libpng freetype glew
 
 - Windows:
 ```
-
+```
```
Comment (cschreib) [inline README.md:140]: If you could please add the missing setup command:
```
vcpkg install sfml zlib libpng freetype glew
```

**Comment by cschreib** [comment]
> A lot has changed!

Indeed... I am using this library for a project of mine, and I need decent support for WebAssembly to run in the browser and make easy demos, as well as proper support for high-DPI displays (which are getting increasingly common these days, particularly for mobile devices). I am also planning to release lxgui 2.0 soon, and I'd like to fix all the problems and quirks with the API before the release.

Anyway, I hope it won't be too annoying for you to upgrade to the latest version. Don't hesitate to ask if you need assistance with that. Once the 2.0 release is out, the API will remain stable.

File: README.md (hunk 1 above)
Comment (wgahnagl) [reply README.md:141]: added!

**Comment by cschreib** [comment]
Thank you!

**Comment by wgahnagl** [comment]
so I am having an issue with finding the LXGUI_LIBRARY dirs in the FindLxgui.cmake file. Are those variables I should set myself or are they supposed to be set by the FindLxgui, and failed? 
It might be helpful to take a look at the project you're working on, but if you're not ready to publish it there's no rush!

**Comment by cschreib-ibex** [comment]
It depends on how you install lxgui or integrate it with your project. FYI, lxgui doesn't use any fancy or unusual mechanism for this. It relies on standard CMake behavior for Find* scripts, so the following isn't really specific to lxgui.

The "standard" method would be to build lxgui this way
```
cd lxgui
mkdir build
cd build
cmake ../
make
sudo make install
```
This will install lxgui in your system folder. If you don't want to modify the system folders, or don't have sudo access, the alternative is
```
cd lxgui
mkdir build
cd build
cmake ../ -DCMAKE_INSTALL_PREFIX=/home/wgahnagl/some_directory
make
make install
```
This will install lxgui in `/home/wgahnagl/some_directory`.

Then, for your own project. If you installed lxgui in the system folder, all you have to do is call 
```
find_package(lxgui 2)
```
in your CMake script. This should find the library automatically without any other input, and define all the `LXGUI_LIBRARIES` (etc) CMake variables that you can use. If instead you installed lxgui in a custom folder, you need to give CMake a hint to the path where lxgui is located. The recommended way is to do it on the command line when invoking CMake:
```
cmake ../ -DLXGUI_ROOT=/home/wgahnagl/some_directory
```
Alternatively you can also just define an environment variable `LXGUI_ROOT` with the value `/home/wgahnagl/some_directory`.

**Comment by cschreib** [comment]
It depends on how you install lxgui or integrate it with your project. FYI, lxgui doesn't use any fancy or unusual mechanism for this. It relies on standard CMake behavior for Find* scripts, so the following isn't really specific to lxgui.

The "standard" method would be to build lxgui this way
```
cd lxgui
mkdir build
cd build
cmake ../
make
sudo make install
```
This will install lxgui in your system folder. If you don't want to modify the system folders, or don't have sudo access, the alternative is
```
cd lxgui
mkdir build
cd build
cmake ../ -DCMAKE_INSTALL_PREFIX=/home/wgahnagl/some_directory
make
make install
```
This will install lxgui in `/home/wgahnagl/some_directory`.

Then, for your own project. If you installed lxgui in the system folder, all you have to do is call 
```
find_package(lxgui 2)
```
in your CMake script. This should find the library automatically without any other input, and define all the `LXGUI_LIBRARIES` (etc) CMake variables that you can use. If instead you installed lxgui in a custom folder, you need to give CMake a hint to the path where lxgui is located. The recommended way is to do it on the command line when invoking CMake:
```
cmake ../ -DLXGUI_ROOT=/home/wgahnagl/some_directory
```
Alternatively you can also just define an environment variable `LXGUI_ROOT` with the value `/home/wgahnagl/some_directory`.

**Pull request merged**